<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-E57MT6J" failures="1" tests="9" name="Test: HomePage and Order" time="105.833" errors="0" timestamp="2020-04-03T01:05:14 CEST">
  <testcase classname="tests.HomePageTest" name="test01WomenBtn" time="9.214"/>
  <testcase classname="tests.HomePageTest" name="test02DressBtn" time="6.775"/>
  <testcase classname="tests.HomePageTest" name="test03SameURL" time="12.287"/>
  <testcase classname="tests.Summer_DressesTest" name="test01_clickFirst" time="5.009"/>
  <testcase classname="tests.Summer_DressesTest" name="test02_Order" time="2.606"/>
  <testcase classname="tests.Summer_DressesTest" name="test03_SizeColor" time="7.953"/>
  <testcase classname="tests.Summer_DressesTest" name="test07_register" time="2.266"/>
  <testcase classname="tests.Summer_DressesTest" name="test08_register" time="34.383"/>
  <testcase classname="tests.Summer_DressesTest" name="test09_registerBack" time="5.389">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [2] but found [(empty)]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2] but found [(empty)]
at tests.Summer_DressesTest.test09_registerBack(Summer_DressesTest.java:146)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test09_registerBack -->
  <testcase classname="tests.Summer_DressesTest" name="test04_Qty" time="0.0">
    <ignored/>
  </testcase> <!-- test04_Qty -->
  <testcase classname="tests.Summer_DressesTest" name="test06_proceedBtn" time="0.0">
    <ignored/>
  </testcase> <!-- test06_proceedBtn -->
  <testcase classname="tests.Summer_DressesTest" name="test05_Title" time="0.0">
    <ignored/>
  </testcase> <!-- test05_Title -->
</testsuite> <!-- Test: HomePage and Order -->
